edx_django_utils/admin/manage_users_by.py

```diff
@@ -0,0 +1,142 @@
+"""
+Management command `manage_user` is used to idempotently create or remove
+Django users, set/unset permission bits, and associate groups by name.
+"""
+
+from logging import getLogger
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.hashers import identify_hasher, is_password_usable
+from django.contrib.auth.models import Group
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from django.utils.translation import gettext as _
+
+logger = getLogger(__name__)  # pylint: disable=invalid-name
+
+
+try:
```

Comment:
Hey @awais786, thanks for putting this PR up!  I and Saad agree that the try catch statement is easier than writing a `post_save` signal handler, though it's not clear to me what the pros and cons are.

Do you think we can use the same technique for the generate password logic or somehow have an if statement that only hits that logic if in an edxapp app?

https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/management/commands/manage_user.py#L100-L110

e.g. 

```
try:
    from openedx.core.djangoapps.user_authn.utils import generate_password
except ImportError:
    UserProfile = None
```

Reply:
@adzuci  Sorry I missed this msg. 
Yes you can do this as well. I don't see any issue. Its already implemented at several places. Please check this link.
https://github.com/edx/edx-enterprise/blob/3215aa03dfcee02cbb6d795fb29e025b4656398a/enterprise/utils.py#L45